Tests/Marketplace/marketplace_services.py

```diff
@@ -3007,7 +3026,212 @@ def copy_author_image(self, production_bucket, build_bucket, images_data, storag
             logging.info(f"No added/modified author image was detected in {self._pack_name} pack.")
             return True
 
-    def upload_images(self, index_folder_path, storage_bucket, storage_base_path, diff_files_list, override_all_packs):
+    def copy_readme_images(self, production_bucket, build_bucket, images_data, storage_base_path,
+                           build_bucket_base_path):
+        """ Copies pack's readme_images from the build bucket to the production bucket
+
+        Args:
+            production_bucket (google.cloud.storage.bucket.Bucket): The production bucket
+            build_bucket (google.cloud.storage.bucket.Bucket): The build bucket
+            images_data (dict): The images data structure from Prepare Content step
+            storage_base_path (str): The target destination of the upload in the target bucket.
+            build_bucket_base_path (str): The path of the build bucket in gcp.
+        Returns:
+            bool: Whether the operation succeeded.
+        """
+        task_status = True
+        err_msg = f"Failed copying {self._pack_name} pack integrations images."
```

Comment:
Didn't you mean this?
```suggestion
        err_msg = f"Failed copying {self._pack_name} pack readme images."
```